_rules/6cbcbe.md

```diff
@@ -0,0 +1,279 @@
+---
+name: Links with identical accessible names serve equivalent purpose
+test_type: atomic
+
+description: |
+  This rule checks that links with identical accessible names resolve to the same resource or equivalent resources.
+
+success_criterion: 
+- 2.4.9 # Link Purpose (Link Only)
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Anne Thyme Nørregaard
+---
+
+## Test Procedure
+
+### Applicability
+
+This rule applies to any set of any two or more HTML or SVG elements that have the [semantic role](#semantic-role) of `link`, or a role that inherits from the `link` role, are [included in the accessibility tree](#included-in-the-accessibility-tree), and that have [matching](#matching-characters) [accessible names](#accessible-name) that does not only consist of [whitespace](#whitespace).
+
+**Note:** The test target for this rule is the full set of link elements that share the same [matching](#matching-characters) [accessible name](#accessible-name).
+
+### Expectation
+
+When followed, the links in each set of target elements resolve to the [same resource](#same-resource) or to [equivalent resources](#equivalent-purpose).
+Resolving the links includes potential redirects, if the redirects happen instantly.
```

Comment:
I'm interesting in understanding how a redirection that is not instant can impact the evaluation of how the ressources are the same or not ? 

I see [the discussion here](https://github.com/act-rules/act-rules.github.io/pull/220#discussion_r271721303), and I think it's fair on a user impact side, but i would failed the redirection itself with SC 3.2.5 not on link purpose.

Reply:
Continuing from that example, if `/foo/b.html` redirects to `/foo/a.html` after 10 seconds, a user would actually get to experience and possibly interact with `/foo/b.html` before being redirected to `/foo/a.html`. As such, `/foo/b.html` is experienced as distinct from `/foo/a.html` and the two are therefore clearly not the same resource. If, on the other hand, `/foo/b.html` redirects instantly to `/foo/a.html`, the two are experienced as the same resource, namely `/foo/a.html`.

I absolutely agree that such a delayed redirect should fail the WCAG, but that is not the concern of this rule. As you point out, it would also fail this rule which may or may not be appropriate; I don't know enough about the WCAG to have an opinion on that though.

Reply:
Thank you @kasperisager for the explanation. 

@WilcoFiers your comments on putting it as a note means that the redirection is not to be tested within that rule?